app/waf/src/models/adc.model.ts

```diff
@@ -148,11 +176,86 @@ export class Adc extends CommonEntity {
   })
   status: ConfigTypes['status'];
 
-  // @property({
-  //   type: 'object',
-  //   required: true,
-  // })
-  // onBoarding: ConfigBody['onBoarding'];
+  @property({
+    type: 'object',
+    schema: {
+      create: true,
```

Comment:
I do not think API can allow to input this property.

Reply:
Done. removed it as we discussed in the sync up meeting.

Reply:
Thanks